cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,49 @@
 /// <reference types='cypress' />
+import { dataForRegister } from "../support/dataForRegister";
+import LogIn from "../support/pages/LogIn";
+import Modal from "../support/pages/Modal";
 
 describe('Student Registration page', () => {
   before(() => {
-    
+    cy.visit("/")
   });
 
-  it('', () => {
-    
+  it('Can registraite with valid data', () => {
```

Comment:
registraite => register 
It would be nice to add full summary "The user can register with valid data"
Please check everywhere